### ISSUE TITLE
CRF module

### ISSUE THREAD

**Issue by Smile-L-up** [microsoft/onnxruntime#11259]
Hello, I would like to ask you a question.  Whether the ONNX engine currently supports CRF module.  When I solve NER task, I use CRF_LOG_LIKELIHOOD to calculate the loss, write it in the call function of the model, and return logits, log_likelihood, transition_params. The generated model can be converted into ONNX model, but it cannot be loaded.  When I don't write the CRF_LOG_LIKELIHOOD in the model, I only return the logits of the network.  It can load and reason normally.  However, there is a problem that only return logits part to ONNX reasoning time, than directly return logits, log_likelihood, transition_params model reasoning time.  Therefore, I wonder if ONNX does not support CRF or if it is my problem.  So far, based on my tests, it's faster to opt out of ONNX.

**Comment by pranavsharma** [comment]
Can you please fill out the issue filing template so that we can take a more informed look?

**Comment by Smile-L-up** [comment]
> Can you please fill out the issue filing template so that we can take a more informed look?
Hello, I seem to have solved my problem. I will close this question later. Thank you for your reply.

**Comment by MrRace** [comment]
@Smile-L-up How do you solved the problem?

**Comment by Smile-L-up** [comment]
I did it by putting CRF module outside the model,  and calculating the losses outside the model.
